bin/pygrb/pycbc_pygrb_plot_coh_ifosnr

```diff
@@ -52,54 +53,75 @@ __program__ = "pycbc_pygrb_plot_coh_ifosnr"
 # =============================================================================
 # Functions
 # =============================================================================
-# Function to load necessary SNR data from a trigger/injection file
-def load_data(input_file, vetoes, ifos, opts, injections=False):
-    """Build a dictionary containing SNR and sensitivity data extracted
-    from a trigger/injection file"""
+# Function to load trigger data
+def load_data(input_file, vetoes, ifos, opts):
+    """Load data from a trigger/injection file"""
+    logging.info("Loading triggers...")
 
-    # Inizialize the dictionary
+    # Initialize the dictionary
     data = {}
-    data['snr'] = None
-    data['ifo_snr'] = None
+    data['coherent'] = None
```

Comment:
You can remove this and simply assign `trigs['network/coherent_snr'][:])` to ` data['coherent']` later